zanata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<config xmlns="http://zanata.org/namespace/config/">
+  <url>https://translate.engineering.redhat.com/</url>
+  <project>CloudForms_UI</project>
+  <project-version>4.0</project-version>
+  <project-type>podir</project-type>
+
+  <locales>
+    <locale>ru</locale>
+    <locale>zh-CN</locale>
+    <locale>ja</locale>
```

Comment:
I don't know if this is example or not, but this does not match the files we have.